pkg/api/v1/boot.proto

```diff
@@ -0,0 +1,131 @@
+syntax = "proto3";
+
+package api.v1;
+
+option go_package = "./api/v1";
+
+service BootService {
+    // Dhcp is the first dhcp request (option 97). A ProvisioningEventPXEBooting is fired
+    rpc Dhcp(BootServiceDhcpRequest) returns (BootServiceDhcpResponse) {}
```

Comment:
Maybe it makes sense to directly to call the event endpoint?

Reply:
No, i wanted to be explicit about whats going on